package.json

```diff
@@ -105,10 +105,10 @@
 	},
 	"dependencies": {
 		"@changesets/cli": "^2.26.2",
+		"@guardian/ophan-tracker-js": "2.0.4",
```

Comment:
The carat has been removed. Was this change intentional?

Reply:
Yes I think its good to fix the versions so we have predictable builds. Also makes it easier to keep deps aligned with our consumers. DCR has switched to fixed versions.